pkg/controller/controllercmd/builder.go

```diff
@@ -66,10 +73,40 @@ func (b *ControllerBuilder) WithFileObserver(reactorFunc func(file string, actio
 		}
 		b.fileObserver = observer
 	}
+	reactorFunc := func(filename string, action fileobserver.ActionType) error {
+		message := fmt.Sprintf("Restart triggered because %s", action.String(filename))
```

Comment:
nit: "because %s changed" or "because of %s"

Reply:
fixed.